modules/mauth-common/src/main/java/com/mdsol/mauth/MAuthRequest.java

```diff
@@ -22,15 +27,27 @@
   private final String httpMethod;
   private final long requestTime;
   private final String resourcePath;
+  private final String queryParameters;
+  private final boolean disableV1;
```

Comment:
Field `disableV1` seems to be assigned but never accessed, is it really needed?

Reply:
Please see refactor https://github.com/mdsol/mauth-jvm-clients/pull/89/commits/d24aad04e50a64dc504d8035d8a1a56901a81c5e